vllm/distributed/parallel_state.py

```diff
@@ -471,16 +487,16 @@ def send_object(self, obj: Any, dst: int) -> None:
 
         # Send object size
 
-        torch.distributed.send(size_tensor,
-                               dst=self.ranks[dst],
-                               group=self.cpu_group)
+        handle_size = torch.distributed.isend(size_tensor,
```

Comment:
Did you still see the performance gain with async send/recv? I remember last time you said it does not help much on perf.